micromamba/tests/test_remove.py

```diff
@@ -47,10 +47,7 @@ def test_remove_orphaned(tmp_home, tmp_root_prefix, tmp_xtensor_env, tmp_env_nam
     assert keys.issubset(set(res.keys()))
     assert res["success"]
 
-    if sys.platform == "darwin" and platform.machine() == "arm64":
-        assert len(res["actions"]["UNLINK"]) == 12
-    else:
-        assert len(res["actions"]["UNLINK"]) == 11
+    assert len(res["actions"]["UNLINK"]) == 12
```

Comment:
Yep I'm in favor of removing the platform cases, in 1.x this was fixed with:
```suggestion
    assert len(res["actions"]["UNLINK"]) > 1
```
because that number is not stable...